src/engraving/compat/midi/compatmidirender.cpp

```diff
@@ -327,11 +327,12 @@ void CompatMidiRender::renderTremolo(Chord* chord, std::vector<NoteEventList>& e
     }
 
     // render tremolo with multiple events
+    int divider = 1 << (tremolo.lines() + chord->durationType().hooks());
+    int t = 1;
+    if (divider != 0) {
```

Comment:
Afaics, `divider` can't really be zero, so this check is redundant. 
The only thing we _might_ want to check is that `(tremolo.lines() + chord->durationType().hooks())` is not negative, because a shift operator with a negative RHS is undefined behaviour. But I think even that is not necessary, because we can be certain enough that it won't be negative.